projects/transformers/experiments/finetuning.py

```diff
@@ -108,9 +161,13 @@
     do_train=True,
     do_eval=True,
     do_predict=False,
+    eval_steps=10,
+    evaluation_strategy="steps",
+    load_best_model_at_end=True,
```

Comment:
I might be wrong, but the goal is to evaluate at the end using the best model instead of using the model at the last step (which as we've seen in some finetuning tasks it is usually not the model at the last step due to overfitting).

Reply:
It looks for the checkpoint with the best metric_for_best_model, and loads that model after training, in this case on a finetuning task. It uses that model (as opposed to one saved at a later checkpoint) for evaluation or prediction steps next. load_best_model_at_end is also required if you are using EarylStoppingCallback.